### ISSUE TITLE
[Feature Request] Add "Apply hue/saturation after scan" to Profile Settings

### ISSUE THREAD

**Issue by githubuser181226** [cyanfish/naps2#499]
**Is your feature request related to a problem? Please describe.**
When working with signed or stamped documents I found out I very often need to bump up the saturation to make the handwritten / stamped elements more readable. NAPS2 can apply Brightness and Contrast on each scan already, but not Hue nor Saturation.

**Describe the solution you'd like**
In "Profile Settings" window please, add controls for image "Saturation" and "Hue", same as for "Brightness" and "Contrast",
and in "Advanced Profile Settings" window in "Compatibility" section add "Apply hue/saturation after scan" option.

Alternatively, please, make it so NAPS2 remembers previous applied adjustment settings you can do using the "Image" button, ie. crop, brightness, contrast, hue, saturation, black and white, sharpen, split.

Thank you.

**Comment by Hermholtz** [comment]
Yes, it would be very helpful to have this. I mean to apply those transformations (sharpness and others - just like we have deskew, brightness and contrast now) automatically after scanning a page.

I think it would be best to have them set on the scanning profile just like we have brightness and contrast. Since they're already implemented in the preview window, maybe it's not that hard to add them to the profile.

[EDIT]: I just discovered the goodies that "Document correction" does for the page. It's amazing! Therefore it would be beneficial to have it in the profile as well.

And the absolute boss would be to also have a context menu after selecting the subset of pages to be able to apply those corrections to all selected pages at once - just like a preview window, but for many pages.

Fingers crossed for that! Thanks in advance.
